src/cdp-proxy/reactNativeCDPProxy.ts

```diff
@@ -10,18 +10,30 @@ import {
     IProtocolSuccess
 } from "vscode-cdp-proxy";
 import { URL } from "url";
+import { isVerboseLogLevel, LogLevel } from "../extension/log/LogHelper";
+import { OutputChannelLogger } from "../extension/log/OutputChannelLogger";
 import { IncomingMessage } from "http";
 
-export class RnCdpProxy {
+export class ReactNativeCDPProxy {
     private server: Server | null;
     private hostAddress: string;
     private port: number;
     private debuggerTarget: Connection;
     private applicationTarget: Connection;
+    private outputChannelLogger: OutputChannelLogger;
+    private logLevel: LogLevel;
 
-    constructor(port = 13602, hostAddress = "127.0.0.1") {
+    constructor(port: number, hostAddress: string, logLevel?: string) {
         this.port = port;
         this.hostAddress = hostAddress;
+
+        if (isVerboseLogLevel(logLevel)) {
```

Comment:
Could it be better check it this way?
`logLevel === LogLevel.Verbose`

Reply:
`isVerboseLogLevel` function was removed. Now LogLevel for CDP Proxy is determined [here](https://github.com/microsoft/vscode-react-native/blob/c3400dc7306165bfcf73af591c9c096f1a1894a9/src/debugger/rnDebugSession.ts#L191-L198).